tests/framework/checks/expects.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package checks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	workerLabel                                  = "node-role.kubernetes.io/worker"
+	descriptionTwoWorkerNodesWCPUManagerRequired = "at least two worker nodes with cpumanager are required for migration"
+)
+
+func ExpectAtLeastTwoWorkerNodesWithCPUManager(virtClient kubecli.KubevirtClient) {
+	ExpectAtLeastTwoNodes(GetWorkerNodesWithCPUManagerEnabled(virtClient), "")
+}
+
+// ExpectAtLeastTwoNodes uses gomega.Expect to verify that the given node list contains at least two elements. The
+// optionalDescription can be used to override the default description that notifies the reader of "not enough worker
+// nodes with cpumanager"
+func ExpectAtLeastTwoNodes(nodes []v1.Node, optionalDescription string) bool {
+	if optionalDescription == "" {
+		optionalDescription = descriptionTwoWorkerNodesWCPUManagerRequired
+	}
+	ginkgo.By("ensuring at least 2 worker nodes have cpumanager")
+	return gomega.Expect(len(nodes)).To(gomega.BeNumerically(">=", 2), optionalDescription)
+}
```

Comment:
IMHO the file should not give preference to any specific check. It should be agnostic.
The optionalDescription in currently never provided, and also, the `ginkgo.By` sentence ignores it.
This is what I suggest for this piece of code:
- make the description mandatory since the function just says `ExpectAtLeastTwoNodes`

Let me know what do you think. Thanks
```suggestion
const (
	workerLabel                                  = "node-role.kubernetes.io/worker"
	DescriptionTwoWorkerNodesWCPUManagerRequired = "at least two worker nodes with cpumanager are required for migration"
)

func ExpectAtLeastTwoWorkerNodesWithCPUManager(virtClient kubecli.KubevirtClient) {
	ExpectAtLeastTwoNodes(GetWorkerNodesWithCPUManagerEnabled(virtClient), DescriptionTwoWorkerNodesWCPUManagerRequired)
}

// ExpectAtLeastTwoNodes uses gomega.Expect to verify that the given node list contains at least two elements.
func ExpectAtLeastTwoNodes(nodes []v1.Node, description string) bool {
	ginkgo.By("ensuring at least 2 nodes")
	return gomega.Expect(len(nodes)).To(gomega.BeNumerically(">=", 2), description)
}
```

Reply:
Great suggestion, makes sense, thanks!